user/python-semver/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-semver"
+pkgver = "3.0.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+checkdepends = ["python-pytest"]
+depends = ["python"]
+pkgdesc = "Python package to work with Semantic Versioning"
+maintainer = "Julie Koubova <julie@koubova.net>"
+license = "BSD-3-Clause"
+url = "https://github.com/python-semver/python-semver"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
```

Comment:
python packages should in general use PYPI_SITE (see others, ditto for your other PRs)